arcgis-ios-sdk-samples/Layers/Display subtype feature layer/README.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "category": "Features",
+    "category": "Layers",
```

Comment:
As you can see in the file path, this sample is part of the `Layers` category, instead of `Features`.

The issue is fixed in metadata.